hive/datadog_checks/hive/data/conf.yaml.example

```diff
@@ -480,28 +480,28 @@ init_config:
             alias: hive.server.memory.heap.committed
             metric_type: gauge
     - include:
-        bean: metrics:name=memory.non-heap.init
+        bean: metrics:name=memory.non_heap.init
```

Comment:
Bean name shouldn't be changed, we have no choice here. Only the alias should be changed